tests/testthat/test-repo.R

```diff
@@ -56,6 +56,17 @@ test_that("repo_source() truncates automatically", {
   })
 })
 
+test_that("repo_source() is robust to trailing slash", {
+  pkg <- list(repo = repo_meta_gh_like("https://github.com/r-lib/pkgdown"))
+
+  dummy_meta <- function(x) {
```

Comment:
```suggestion
  meta <- function(x) {
```

https://itconnect.uw.edu/guides-by-topic/identity-diversity-inclusion/inclusive-language-guide/